ПЕ-21б Поздеева О.В.: Добавление метода Create в класс AuditoryReposi…

OxanaPozdeeva opened the pull request:
…tory

Добавила метод Create в класс AuditoryRepository
Написала тесты для этого метода

OxanaPozdeeva committed:
ПЕ-21б Поздеева О.В.: Добавление метода Create в класс AuditoryRepository

Добавила метод Create в класс AuditoryRepository
Написала тесты для этого метода

dha01 commented on IS.Model/IS.Model.Tests/Repository/Auditory/AuditoryRepositoryTests.cs:
```diff
@@ -0,0 +1,129 @@
+        /// <summary>
+        /// Проверяет эквивалентны ли две аудитории.
+        /// </summary>
+        /// <param name="first_auditory"></param>
```
Добавить описание входных параметров.

OxanaPozdeeva committed:
ПЕ-21Б Поздеева О.В.: Создание метода Create в класс AuditoryRepository

Применила табуляцию
Включила в проект AuditoryRepositoryTests.cs
Собрала проект и исправила ошибку

OxanaPozdeeva committed:
ПЕ-21б Поздеева О.В.: Создание тестов для AuditoryRepository

OxanaPozdeeva committed:
ПЕ-21б Поздеева О.В.: Создание метода Create в классе AuditoryRepository

Окончательно измененные файлы

dha01 committed:
Merge branch 'master' into feature/Task_49_Pozdeeva

# Conflicts:
#	IS.Model/IS.Model.Tests/IS.Model.Tests.csproj
#	IS.Model/IS.Model.Tests/Repository/Auditory/AuditoryRepositoryTests.cs

dha01 committed:
Уманов А.А.: название метода для сравнения изменено.

dha01 committed:
Уманов А.А.: отступ
